stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -250,7 +268,12 @@ public class ChannelListViewModel(
         }
 
         channelsState = channelsState.copy(isLoadingMore = true)
-        chatDomain.queryChannelsLoadMore(filter, currentConfig.querySort).enqueue()
+        if (ToggleService.isEnabled(ToggleService.TOGGLE_KEY_OFFLINE)) {
+            queryChannelsState.nextPageRequest.value?.copy(filter = filter, querySort = currentConfig.querySort)
```

Comment:
We should use a nullcheck here in general and chain them.

In case the value is null, we can return, e.g.;

```
val newQueryState = queryChannelsState?.nextPageRequest?.value?.copy(filter = filter, querySort = ...) ?: return
```

Reply:
And also should be using `createQueryChannelsFilter`